SWLOR.Game.Server/Feature/AbilityDefinition/OneHanded/BackstabAbilityDefinition.cs

```diff
@@ -0,0 +1,130 @@
+﻿//using Random = SWLOR.Game.Server.Service.Random;
+using System.Collections.Generic;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core.NWScript.Enum.Creature;
+using SWLOR.Game.Server.Core.Bioware;
+using SWLOR.Game.Server.Enumeration;
+using SWLOR.Game.Server.Service;
+using SWLOR.Game.Server.Service.AbilityService;
+using static SWLOR.Game.Server.Core.NWScript.NWScript;
+using SWLOR.Game.Server.Core.NWScript.Enum.VisualEffect;
+
+namespace SWLOR.Game.Server.Feature.AbilityDefinition
+{
+    public class BackstabAbilityDefinition : IAbilityListDefinition
+    {
+        public Dictionary<Feat, AbilityDetail> BuildAbilities()
+        {
+            var builder = new AbilityBuilder();
+            Backstab1(builder);
+            Backstab2(builder);
+            Backstab3(builder);
+
+            return builder.Build();
+        }
+
+        private static string Validation(uint activator, uint target, int level)
+        {
+            var weapon = GetItemInSlot(InventorySlot.RightHand);
+
+            if (Item.FinesseVibrobladeBaseItemTypes.Contains(GetBaseItemType(weapon))
+                && (GetBaseItemType((GetItemInSlot(InventorySlot.LeftHand))) == Core.NWScript.Enum.Item.BaseItem.SmallShield ||
```

Comment:
Just wondering about the checks for shields. What was your thinking on those?

Reply:
We probably should add a list of shields to the Item service come to think of it.